sdk/postgresql/Azure.ResourceManager.PostgreSql/src/PostgreSqlFlexibleServers/Custom/Models/PostgreSqlBaseCapability.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+namespace Azure.ResourceManager.PostgreSql.FlexibleServers.Models
+{
+    /// <summary> Base object for representing capability. </summary>
+    public partial class PostgreSqlBaseCapability
+    {
+        /// <summary> Readonly status of the capability. </summary>
+        public string? Status
```

Comment:
We do not need the question mark since nullable has been disabled.
```suggestion
        public string Status
```

Reply:
Also since this has been replaced by CapabilityStatus, could we add `[EditorBrowsable(Never)]` to this property?